private/bufpkg/bufplugin/bufpluginconfig/bufpluginconfig.go

```diff
@@ -281,6 +295,13 @@ type ExternalNPMRegistryConfig struct {
 	ImportStyle string `json:"import_style,omitempty" yaml:"import_style,omitempty"`
 }
 
+// ExternalMavenRegistryConfig is the external registry configuration for a Maven plugin.
+type ExternalMavenRegistryConfig struct {
+	Deps []struct {
```

Comment:
What do you think about simplifying this in the config to just use `[]string`? It isn't extensible in the future but it results in a nicer config file IMO.

Reply:
so instead of:

```yaml
registry:
  maven:
    deps:
      - gav: "io.grpc:grpc-core:1.52.1"
      - gav: "io.grpc:grpc-protobuf:1.52.1"
      - gav: "io.grpc:grpc-stub:1.52.1"
```

you'd have:

```yaml
registry:
  maven:
    - gav: "io.grpc:grpc-core:1.52.1"
    - gav: "io.grpc:grpc-protobuf:1.52.1"
    - gav: "io.grpc:grpc-stub:1.52.1"
```
?

I think I lean a little towards the former for congruity with the existing registry specs and the extensibility (which might come into play shortly with the `lite` runtime?).

Reply:
No I was suggesting:

```yaml
registry:
  maven:
    deps:
      - "io.grpc:grpc-core:1.52.1"
      - "io.grpc:grpc-protobuf:1.52.1"
      - "io.grpc:grpc-stub:1.52.1"
```

Reply:
oh! yeah definitely, that's better.

Reply:
done in [`ea99592` (#1801)](https://github.com/bufbuild/buf/pull/1801/commits/ea995926af44faddb9391d66c6f5fa14c21ba274)